app/forms/OD2/generic_form_behavior.rb

```diff
@@ -0,0 +1,23 @@
+module OD2
+  module GenericFormBehavior
+    extend ActiveSupport::Concern
+
+    self.terms += [:alternative, :caption_title, :tribal_title, :arranger, :artist, :author, :cartographer, :collector, :composer, :creator_display, :dedicatee, :donor, :designer, :editor, :former_owner, :illustrator, :interviewee, :interviewer, :lyricist, :owner, :patron, :photographer, :print_maker, :recipient, :scribe,
```

Comment:
I definitely think we need to come up with an elegant solution to a huge term block here. First off I know that @revgum would want it alphabetized. We'll also need to figure out which fields need to go "above the fold" and which "below the fold".